README.md

```diff
@@ -19,21 +18,15 @@
     - [Key Features](#key-features)
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#deployment)
+  
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
```

Comment:
- [ ] Please adhere to the project requirement properly, as I mentioned in my previous review. Last time, you had everything perfect in your README file, but I think you messed up with it. Kindly fix it because the table is just like the `index` of the book where a user can directly jump over a particular section. 

<table>
<tr>
<th align=“middle”>Your README</th>
<th align=“middle”>Professional README</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-30 23-16-30](https://github.com/Afsheen96/Hello-Microverse/assets/66476329/5ed9cf29-2b17-4f0d-87d0-1f715fba2b6f)



</td>
<td>

![image](https://github.com/Afsheen96/Hello-Microverse/assets/66476329/a67a35db-916b-483f-b9c5-78fd526c6bab)

</td>
</tr>
</table>

Reply:
you want me to also add FAQ(OPTIONAL) section? @skyv26